src/test/java/elizCurtnazar/tests/BothPagesTests.java

```diff
@@ -0,0 +1,34 @@
+package elizCurtnazar.tests;
+
+import elizCurtnazar.driver.BrowserManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BothPagesTests {
+    public static void main(String[] args) {
+        ChromeDriver driver = BrowserManager.createDriver();
+        TestPagesHerokuappManager herokuapp = new TestPagesHerokuappManager("https://testpages.herokuapp.com/styled/index.html", driver);
```

Comment:
Move this line after demoqa methods and before the heroku methods in order to have functional run. Thanks!

Reply:
Done.